docs/source/include/learning.rst

```diff
@@ -0,0 +1,93 @@
+Learned Models
+==============
+
+In SCICO, neural network models are used to represent imaging problems and provide different modes of data-driven regularization.
+The models are implemented in `FLAX <https://flax.readthedocs.io/>`_, and constitute a representative sample of frequently used networks.
+
+
+FlaxMap
+-------
+
+SCICO interfaces with the implemented models via :class:`.FlaxMap`. This provides a standardized access to all trained models via the model definiton and the learned parameters. Further specialized functionality, such as learned denoisers, are built on top of :class:`.FlaxMap`. The models implemented are described next.
+
```

Comment:
Suggested: `The specific models that have been implemented are described below`.

Reply:
Changed to the suggested form.